packages/venia-ui/lib/components/Gallery/item.js

```diff
@@ -1,138 +1,113 @@
-import React, { Component } from 'react';
+import React from 'react';
 import { string, number, shape } from 'prop-types';
 import { Link, resourceUrl } from '@magento/venia-drivers';
 import { Price } from '@magento/peregrine';
-
-import classify from '../../classify';
 import { transparentPlaceholder } from '@magento/peregrine/lib/util/images';
-import { generateSrcset } from '../../util/images';
+
+import { mergeClasses } from '../../classify';
+import Image from '../Image';
 import defaultClasses from './item.css';
 
 // The placeholder image is 4:5, so we should make sure to size our product
 // images appropriately.
 const imageWidth = '300';
 const imageHeight = '375';
 
-const ItemPlaceholder = ({ children, classes }) => (
-    <div className={classes.root_pending}>
-        <div className={classes.images_pending}>{children}</div>
-        <div className={classes.name_pending} />
-        <div className={classes.price_pending} />
-    </div>
-);
+// See gallery.css for breakpoints and number of columns.
+const galleryBreakpoint = '640px';
```

Comment:
Also, are these values always constant? I am trying to see if they can be moved to a separate file of their own, something like `constants/image.js` and fetched from there so it will be easy to maintain and use for potential vendors.

Reply:
Good idea. They've since been altered slightly but these are coupled to this specific component so I don't think a separate file would help much.